Volume/Views/MainView and Tabs/BookmarksView.swift

```diff
@@ -183,3 +213,9 @@ struct BookmarksView: View {
     }
     
 }
+
+//struct BookmarksView_Previews: PreviewProvider {
+//    static var previews: some View {
```

Comment:
Is there a reason why this code is commented out instead of deleted. If there is a reason, add a mark or TODO

Reply:
It's for live preview, but I guess I could add a comment